CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.7.0 (2017-01-05)
+feat: update to http-ece 0.6.4 (with draft-06 support)
```

Comment:
0.6.5 came out after this landed. I'll need to verify that 0.6.5 contains no additional changes to the data constructs.